pages/asanas/[id].js

```diff
@@ -0,0 +1,80 @@
+import { getAsanaByID, getAllAsanas } from "../../db";
+import Image from "next/image";
+import styled from "styled-components";
+
+export async function getStaticPaths() {
+  const asanas = getAllAsanas();
+  const ids = asanas.map((asana) => asana.id);
```

Comment:
I guess you want the ids to be 1,2,3,... to have a respective URL, when clicking through the asanas. However, the id in the original array should be a valid id (see comment above). For the URL you could add another attribute to the asana object, like URL_id.

Reply:
That's true. But this is a good suggestion. I think a will do that in a short user story, because it effects several files in the code ;)